packages/react/src/config.ts

```diff
@@ -1,2 +1,8 @@
 export const CONNECTOR_KEY = 'fuel-current-connector';
 export const BRIDGE_URL = 'https://app.fuel.network/bridge?from=eth&to=fuel';
+export const NATIVE_CONNECTORS = [
+  'Fuel Wallet',
+  'Fuel Wallet Development',
+  'Fuelet Wallet',
+  'Bako Safe',
```

Comment:
Missing burner wallet

Reply:
As burner is dosen't have a review tx page, I think we should keep as external